documentation/library_guide/parallel_api/execution_policies.rst

```diff
@@ -1,65 +1,70 @@
+---------------------- -----------------------------------------------------
+``unseq``              The standard aligned policy for unsequenced SIMD execution.
+                       This policy requires user-provided functions to be SIMD-safe.
+---------------------- -----------------------------------------------------
+``par``                The standard aligned policy for parallel execution by multiple threads.
+---------------------- -----------------------------------------------------
+``par_unseq``          The standard aligned policy with the combined effect of ``unseq`` and ``par``.
+---------------------- -----------------------------------------------------
+``device_policy``      The class template to create device policies for data parallel execution.
+---------------------- -----------------------------------------------------
+``dpcpp_default``      The device policy for data parallel execution on the default SYCL device.
+---------------------- -----------------------------------------------------
+``fpga_policy``        The class template to create policies for execution on FPGA devices.
+---------------------- -----------------------------------------------------
+``dpcpp_fpga``         The device policy for data parallel execution on a SYCL FPGA device.
+====================== =====================================================
 
 The implementation is based on Parallel STL from the
 `LLVM Project <https://github.com/llvm/llvm-project/tree/main/pstl>`_.
 
 |onedpl_short| supports two parallel backends for execution with ``par`` and ``par_unseq`` policies:
 
-#. TBB backend (enabled by default) uses |onetbb_long| or |tbb_long| for parallel execution.
+#. The TBB backend (enabled by default) uses |onetbb_long| or |tbb_long| for parallel execution.
 
-#. OpenMP backend uses OpenMP* pragmas for parallel execution. Visit
+#. The OpenMP backend uses OpenMP* pragmas for parallel execution. Visit
    :doc:`Macros <../macros>` for the information how to enable the OpenMP backend.
 
+OpenMP pragmas are also used for SIMD execution with ``unseq`` and ``par_unseq`` policies.
+
 Follow these steps to add Parallel API to your application:
 
 #. Add ``#include <oneapi/dpl/execution>`` to your code.
    Then include one or more of the following header files, depending on the algorithms you
    intend to use:
 
-   #. ``#include <oneapi/dpl/algorithm>``
-   #. ``#include <oneapi/dpl/numeric>``
-   #. ``#include <oneapi/dpl/memory>``
+   - ``#include <oneapi/dpl/algorithm>``
+   - ``#include <oneapi/dpl/numeric>``
+   - ``#include <oneapi/dpl/memory>``
 
-#. Pass a |onedpl_short| execution policy object, defined in the ``oneapi::dpl::execution``
-   namespace, to a parallel algorithm.
-#. Use the C++ standard execution policies:
+#. Pass a |onedpl_short| execution policy object as the first argument to a parallel algorithm
+   to specify the desired execution behavior.
```

Comment:
"desired execution behavior" - I remembered that a host execution policy doesn't  guarantee execution behavior, it may have the desired behavior, it is just a hint. 
For a device policy provides the desired (device execution) execution behavior.
Probably this difference should be explained in the doc.

Reply:
Thanks, I will think of it.

Reply:
I rephrased it as "to indicate the desired behavior" to sound a bit less formal. Anyway, the sentence describes a user action, and the behavior is desired, not guaranteed. Also I added  "possible" to execution descriptions for `unseq` and `par`.